src/migrations/1728032163157-dropAllIndexesFKsRelations.ts

```
@@ -0,0 +1,103 @@
+        await queryRunner.query(
+          `ALTER TABLE \`${fk.TABLE_NAME}\` DROP FOREIGN KEY \`${fk.CONSTRAINT_NAME}\``
+        );
+        console.log(
+          `Dropped foreign key ${fk.CONSTRAINT_NAME} on table ${fk.TABLE_NAME}`
+        );
+      } catch (error) {
+        console.error(
+          `Could not drop foreign key ${fk.CONSTRAINT_NAME} on table ${fk.TABLE_NAME}:`,
+          error
+        );
+        continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)